packages/gatsby-transformer-screenshot/src/gatsby-node.js

```diff
@@ -16,29 +16,30 @@ exports.onPreBootstrap = (
   pluginOptions
 ) => {
   const { createNode, touchNode } = actions
+  const screenshotNodes = Array.from(store.getState().nodes.values()).filter(
```

Comment:
`Array.from(store.getState().nodes.values())` could be replaced by `getNodes` I think?

Reply:
Good shout, thanks. Hmmm... we should add docs for `getNodes()` I guess?

Reply:
Yeah, about all helper functions that get passed to plugins actually